SYCL/ESIMD/lsc/lsc_usm_prefetch_u32_scalar_off.cpp

```diff
@@ -0,0 +1,17 @@
+//==--- lsc_usm_prefetch_u32_scalar_off.cpp - DPC++ ESIMD on-device test ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
```

Comment:
nit: This line is not needed because the previous line requires gpu-intel-pvc || esimd_emulator, which automatically disables the test for cuda|hip.